<ui version="4.0" >
 <class>SystemDialog</class>
 <widget class="QDialog" name="SystemDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>New System</string>
  </property>
  <widget class="QLabel" name="uiLabelInitialSystem" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>12</x>
     <y>316</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Initial System:</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="uiComboInitialSystem" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>111</x>
     <y>316</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>268</x>
     <y>310</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="uiTabsSystem" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>13</y>
     <width>421</width>
     <height>291</height>
    </rect>
   </property>
   <property name="contextMenuPolicy" >
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="currentIndex" >
    <number>0</number>
   </property>
   <widget class="QWidget" name="uiTabSystemGeneral" >
    <attribute name="title" >
     <string>General</string>
    </attribute>
    <widget class="QComboBox" name="uiCombboxSystemFolder" >
     <property name="geometry" >
      <rect>
       <x>125</x>
       <y>60</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable" >
      <bool>true</bool>
     </property>
     <property name="maxVisibleItems" >
      <number>4</number>
     </property>
    </widget>
    <widget class="QPushButton" name="browseButton" >
     <property name="geometry" >
      <rect>
       <x>350</x>
       <y>60</y>
       <width>40</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="uiLabelSystemFolder" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>61</y>
       <width>102</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text" >
      <string>System Folder:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_1" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>20</y>
       <width>102</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>System Name:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="openOnStartupCB" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>231</y>
       <width>281</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text" >
      <string>Open this System Confiruation  on Start-up</string>
     </property>
    </widget>
    <widget class="QLabel" name="uiLabelGeneralLoadFirmware" >
     <property name="geometry" >
      <rect>
       <x>8</x>
       <y>160</y>
       <width>104</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text" >
      <string>Startup Firmware:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="uiLEditGeneralLoadFirmware" >
     <property name="geometry" >
      <rect>
       <x>125</x>
       <y>156</y>
       <width>271</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip" >
      <string>Enter the filename of a DVR firmware to load.</string>
     </property>
     <property name="whatsThis" >
      <string>Enter  the path and name of a DVR firmware to be loaded on the startup of the application. Note that there is only one startup firmware for all the system configuration. The value of this field is kept in the registary and not the config.xml file.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>97</y>
       <width>102</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>Video Standard:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="uiLabelAudioRate" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>128</y>
       <width>102</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text" >
      <string>Audio Rate:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="videoStdCombo" >
     <property name="geometry" >
      <rect>
       <x>125</x>
       <y>97</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip" >
      <string>Select a video standard to be loaded next the system is opened.</string>
     </property>
    </widget>
    <widget class="QComboBox" name="uiComboAudioRate" >
     <property name="geometry" >
      <rect>
       <x>125</x>
       <y>127</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="sysNameEdit" >
     <property name="geometry" >
      <rect>
       <x>125</x>
       <y>20</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="uiTabSystemSettings" >
    <attribute name="title" >
     <string>Settings</string>
    </attribute>
    <widget class="QLineEdit" name="uiLEditSettingsStatInterval" >
     <property name="geometry" >
      <rect>
       <x>347</x>
       <y>69</y>
       <width>49</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maxLength" >
      <number>2</number>
     </property>
    </widget>
    <widget class="QLabel" name="uiLabelMaxH264Frames" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>100</y>
       <width>302</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip" >
      <string/>
     </property>
     <property name="whatsThis" >
      <string>Enter maximum number of frames to record in a H.264 or AVI recorded file. Enter zero for no limit.</string>
     </property>
     <property name="text" >
      <string>Max. Number of Frames to Record:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="uiChkBoxSegmentRec" >
     <property name="geometry" >
      <rect>
       <x>19</x>
       <y>127</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text" >
      <string>Create a New File on Recording.</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="uiChkBoxMJPEGSytleAVI" >
     <property name="geometry" >
      <rect>
       <x>19</x>
       <y>153</y>
       <width>391</width>
       <height>21</height>
      </rect>
     </property>
     <property name="autoFillBackground" >
      <bool>true</bool>
     </property>
     <property name="text" >
      <string>Use AVI style file for MJPEG Recording. (Uncheck to use .mov)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="uiChkBoxSettingsWatchdogTimer" >
     <property name="geometry" >
      <rect>
       <x>19</x>
       <y>220</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text" >
      <string>Enable Watchdog Timer</string>
     </property>
    </widget>
    <widget class="QLabel" name="uiLabelWatchdotTimer" >
     <property name="geometry" >
      <rect>
       <x>186</x>
       <y>220</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>Watchdog Timeout(sec.):</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="uiLEditSettingsWatchTimeout" >
     <property name="geometry" >
      <rect>
       <x>347</x>
       <y>218</y>
       <width>49</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="uiLEditMaxH264Frames" >
     <property name="geometry" >
      <rect>
       <x>347</x>
       <y>99</y>
       <width>49</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label" >
     <property name="geometry" >
      <rect>
       <x>40</x>
       <y>177</y>
       <width>321</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text" >
      <string>(There is no audio support in AVI file recording.)</string>
     </property>
    </widget>
    <widget class="QLabel" name="uiLabelSettingsTimeOut" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>42</y>
       <width>302</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text" >
      <string>PCI Timeout (sec.):</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="uiLEditSettingsTimeOut" >
     <property name="geometry" >
      <rect>
       <x>347</x>
       <y>40</y>
       <width>49</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maxLength" >
      <number>2</number>
     </property>
    </widget>
    <widget class="QLabel" name="uilabelSettingsStatInterval" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>69</y>
       <width>302</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>Frame Rate Statistics Interval (sec.):</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="uiLabelTimeUpdateInterval" >
     <property name="geometry" >
      <rect>
       <x>30</x>
       <y>15</y>
       <width>307</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text" >
      <string>Date/Time Update Interval (minutes):</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="uiLEditTimeUpdateInterval" >
     <property name="geometry" >
      <rect>
       <x>347</x>
       <y>10</y>
       <width>49</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="uiTabSystemDebug" >
    <attribute name="title" >
     <string>Debug</string>
    </attribute>
    <widget class="QGroupBox" name="uiGrpSystemDebugDebug" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>391</width>
       <height>221</height>
      </rect>
     </property>
     <property name="title" >
      <string>Enable Debug</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <widget class="QCheckBox" name="uiChkDebugAVFrames" >
      <property name="geometry" >
       <rect>
        <x>90</x>
        <y>27</y>
        <width>95</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text" >
       <string>A/V Frames</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="uiChkDebugSDK" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>27</y>
        <width>50</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text" >
       <string>SDK</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="uiTabSystemOSDFont" >
    <attribute name="title" >
     <string>OSD Font</string>
    </attribute>
    <widget class="QGroupBox" name="uiGrpUseCustomFont" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>401</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title" >
      <string>Use Custom Font Table:</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="uiLabelColor_y" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>110</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>Y-Color:</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="uiLEditColor_y" >
      <property name="geometry" >
       <rect>
        <x>60</x>
        <y>110</y>
        <width>60</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="uiLabelColor_u" >
      <property name="geometry" >
       <rect>
        <x>130</x>
        <y>110</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>U-Color:</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="uiLEditColor_u" >
      <property name="geometry" >
       <rect>
        <x>200</x>
        <y>110</y>
        <width>60</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="uiLabelColor_v" >
      <property name="geometry" >
       <rect>
        <x>260</x>
        <y>110</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>V-Color:</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="uiLabelFontTableFile" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>30</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text" >
       <string>Font File (.bdf):</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="uiLEditFontFile" >
      <property name="geometry" >
       <rect>
        <x>104</x>
        <y>28</y>
        <width>231</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="uiBtnFontFileBrowse" >
      <property name="geometry" >
       <rect>
        <x>354</x>
        <y>28</y>
        <width>40</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text" >
       <string>...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="uiLEditStartCode" >
      <property name="geometry" >
       <rect>
        <x>102</x>
        <y>68</y>
        <width>60</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="uiLabelStartCode" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>70</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>Start Code:</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="uiLabelEndCode" >
      <property name="geometry" >
       <rect>
        <x>252</x>
        <y>70</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text" >
       <string>End Code:</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="uiLEditColor_v" >
      <property name="geometry" >
       <rect>
        <x>330</x>
        <y>110</y>
        <width>60</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="uiLEditEndCode" >
      <property name="geometry" >
       <rect>
        <x>330</x>
        <y>68</y>
        <width>60</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>uiTabsSystem</tabstop>
  <tabstop>sysNameEdit</tabstop>
  <tabstop>uiCombboxSystemFolder</tabstop>
  <tabstop>browseButton</tabstop>
  <tabstop>videoStdCombo</tabstop>
  <tabstop>uiComboAudioRate</tabstop>
  <tabstop>uiLEditGeneralLoadFirmware</tabstop>
  <tabstop>openOnStartupCB</tabstop>
  <tabstop>uiComboInitialSystem</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>uiLEditEndCode</tabstop>
  <tabstop>uiLEditColor_y</tabstop>
  <tabstop>uiLEditColor_u</tabstop>
  <tabstop>uiLEditColor_v</tabstop>
  <tabstop>uiLEditTimeUpdateInterval</tabstop>
  <tabstop>uiLEditSettingsTimeOut</tabstop>
  <tabstop>uiLEditSettingsStatInterval</tabstop>
  <tabstop>uiLEditMaxH264Frames</tabstop>
  <tabstop>uiChkBoxSegmentRec</tabstop>
  <tabstop>uiChkBoxMJPEGSytleAVI</tabstop>
  <tabstop>uiChkBoxSettingsWatchdogTimer</tabstop>
  <tabstop>uiLEditSettingsWatchTimeout</tabstop>
  <tabstop>uiChkDebugSDK</tabstop>
  <tabstop>uiChkDebugAVFrames</tabstop>
  <tabstop>uiLEditFontFile</tabstop>
  <tabstop>uiBtnFontFileBrowse</tabstop>
  <tabstop>uiLEditStartCode</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SystemDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SystemDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
